components/bms_worker/src/BatteryMonitoring.c

```diff
@@ -417,3 +420,25 @@ void BMS_DischargeLimit(uint16_t relativeSoC) {
         BMS.dischargeLimit = 2.25f*relativeSoC; //linear function for the last 20% of discharge
     }
 }
+
+void BMS_HeatCurrentDischargeLimit(int16_t cellTemp) {
+    if (cellTemp >= 48) {
+        BMS.heatChargeCurrentLimit    = -3.75f * cellTemp + 225;
```

Comment:
im not sure what this formula means

Reply:
from the range of (48-60°C) it represents a linear function like for the standard ccl and dcl, but with temps on the x-axis. So at 48°C it has a current discharge limit of (-3.75*48+225 = 45A), at 60 (max temp --> -3.75*60+225 = 0A). 
--> 48°C chosen because its 20% from 60, (60*0.8=48)